Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -113,26 +92,24 @@ td.formGridTopDataRow
 {
   visibility: hidden;
   transition-property: visibility;
-  transition-duration: 0;
-  transition-delay: 0;
+  transition-duration: 0s;
+  transition-delay: 0s;
   transition-timing-function: linear;
-
   content: attr(aria-label);
   white-space: pre;
   display: inline-block;
   position: absolute;
-  left: 0;
-  top: 1.2em;
-  z-index: 1000;
-  padding: 0.25em;
-  border: 1px solid Black;
-  background-color: Lightyellow;
-  box-sizing: border-box;
+  /* popup spawns at the bottom right corner of the icon's hover highlighting */
+  top: calc(var(--default-icon-size) + var(--remotion-themed-spacing-half));
```

Comment:
Okay, this is because the top starts at the icon's top and then we have to push it down an exrtra half spacing?